github/repos_projects.go

```diff
@@ -10,10 +10,19 @@ import (
 	"fmt"
 )
 
+// ProjectListOptions specifies the optional parameters to the
+// OrganizationsService.ListProjects and RepositoriesService.ListProjects methods.
+type ProjectListOptions struct {
+	// Indicates the state of the projects to return. Can be either open, closed, or all. Default: open
+	State string `url:"state,omitempty"`
```

Comment:
This is normal. `url` struct field tag controls how the field gets marshaled into a query parameter.

Take a look at `addOptions` helper which operates on this struct:

https://github.com/google/go-github/blob/master/github/github.go#L191

Also look at the [`github.com/google/go-querystring/query`](github.com/google/go-querystring/query) package being used.

You can confirm for yourself by removing the `url` struct field tag, and see how the test fails. The query parameter will be encoded as "State" instead of "state" without it.